### DISCUSSION TITLE
Electrolama zig-a-zig-ah! (zzh!) vs slaesh’s CC2652RB stick

### DISCUSSION THREAD

**Discussion by kitopopo** [Koenkk/zigbee2mqtt#5935] [General]
Dear @Koenkk ,

I need upgrade my old cc2531. Now i can see two models. Wich is the diference between both models? Which is better? Can you explain the difference? Thanks.

I use domoticz with the plugin zigbe2mqtt, this plugin works correctly with both models?

Thanks in advanced.

**Comment by Koenkk** [comment]
Wrote some docs about this yesterday, see point 3 of https://github.com/Koenkk/zigbee2mqtt.io/blob/develop/docs/information/supported_adapters.md#notes

TLDR:; no difference

**Reply by kitopopo** [reply]
Ok thanks. Both models can work with zigbe2mqtt plugin?

**Reply by walthowd** [reply]
Small differences -- Slaesh's stick properly identifies itself via the USB serial adapter, which is nice -- It also can be reset and do firmware updates without needing BSL physical button like on the ZZH -- Both the sticks/CC2652s suffer from a as yet unidentified Z-Stack lock up bug but Slaesh's stick could be automatically reset for recovery which is a bug plus.

**Reply by kitopopo** [reply]
> Small differences -- Slaesh's stick properly identifies itself via the USB serial adapter, which is nice -- It also can be reset and do firmware updates without needing BSL physical button like on the ZZH -- Both the sticks/CC2652s suffer from a as yet unidentified Z-Stack lock up bug but Slaesh's stick could be automatically reset for recovery which is a bug plus. 

Hi dear friends,

Both the sticks/CC2652s suffer from a as yet unidentified Z-Stack lock up bug? It is really @Koenkk ?

**Reply by Koenkk** [reply]
Should be fixed in latest firmware (https://github.com/Koenkk/zigbee2mqtt/discussions/6019). https://github.com/Koenkk/Z-Stack-firmware/tree/master/coordinator/Z-Stack_3.x.0/bin

**Reply by kitopopo** [reply]
> Should be fixed in latest firmware (https://github.com/Koenkk/zigbee2mqtt/discussions/6019). https://github.com/Koenkk/Z-Stack-firmware/tree/master/coordinator/Z-Stack_3.x.0/bin

then does the cc2531 stick still work better than the zzh? Do you advise waiting for the firmware of the zzh to be improved? I would like to progress instead of getting worse

**Reply by Koenkk** [reply]
The ZZH firmware has already been updated to fix this, the ZZH is a very big improvement over the CC2531.

**Comment by Urwiss** [comment]
Dont buy from slaesh. No support, no answers, no conntact

**Reply by m0wlheld** [reply]
Wrong.

**Reply by Urwiss** [reply]
Just try to conntact to him via telegram. He is not active more than month. But he wants such a conntact with problems. And see comments in other sites

**Reply by oriolism** [reply]
I bought a CC2652RB stick a week ago and i haven't received any update. Already tried to contact him on telegram and email without success.

**Comment by kitopopo** [comment]
I bought zzh stick some days ago. I bought also the debug kit. I am waiting the shipment. I hope that mr koenkk can give same support in the zzh that with the cc2531 stick. 

Thanks

**Reply by kitopopo** [reply]
Congratulations @Koenkk , I hace received and configured the new zzh and works correctly, 
I will comment later on a couple of problems that I have found to see if we can solve them but in general it works much better and faster than the cc2531. I am glad I made this purchase. my opinion is that your system is more open and configurable than conbee. Thanks for your design

Best regards.
